### ISSUE TITLE
journalctl: Emit exactly one field

### ISSUE THREAD

**Issue by muelli** [systemd/systemd#15621]
Recently, I was hacking something together with a bunch of shell scripts and virtual machines. In the end, I wanted to reach into the guest and extract a certain field (`HELLO_ADDR`) from the journal.
I have copied the journal directory out of the guest and tried various commands:

```
journalctl --output-fields=HELLO_ADDR MESSAGE_ID=ba51ad4293834fa695b4c548971d9a71

journalctl --output=export --output-fields=HELLO_ADDR MESSAGE_ID=ba51ad4293834fa695b4c548971d9a71

journalctl --output=verbose --output-fields=HELLO_ADDR MESSAGE_ID=ba51ad4293834fa695b4c548971d9a71
```
 and so on.

None of those commands gave me the one field I was interested in.
So I ended up using the export, which also spits out some other fields, and grepped my way through the output.
That is not ideal as one of the auxiliary fields (which I am not interested in) *could* contain the name of my field.

I think I would have appreciated an option for rendering my field (and erroring out if none was found), maybe `--single-field` in combination with `--output-list`. I haven't really thought about the UI and I believe that others are in a much better position to come up with a scheme that fits the rest of the ecosystem.

**Comment by poettering** [comment]
makes sense. maye we should have `--output=value-only` or so, which would just output the rvalue of the selected fields, suppressing the lvalue and the `=`. you could then do:

```
journalctl --output=fields-only --output-fields=FOOBAR
```

and `journalctl --output=cat` would then become equivalent to `journalctl --output=fields-only --output-fields=MESSAGE`.

**Comment by poettering** [comment]
Fix in #15648
